src/components/inputs/localized-money-input/README.md

```diff
@@ -171,3 +196,121 @@ LocalizedMoneyInput.hasEmptyValue([
   { amount: '', currencyCode: 'USD' },
 ]); // ->  true
 ```
+
+### Example
+
+Here's an example of how `LocalizedMoneyInput` would be used inside a form.
+
+```jsx
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { Formik } from 'formik';
+import omitEmpty from 'omit-empty';
+import ErrorMessage from '@commercetools-frontend/ui-kit/messages/error-message';
+import { LocalizedMoneyInput } from '@commercetools-frontend/ui-kit';
+
+
+// the existing document, e.g. from the database
+const doc = {
+  prices: [
+    {
+      currencyCode: 'ALL',
+      centAmount: 1200
+    },
+    {
+      currencyCode: 'AMD',
+      centAmount: 3300,
+    }
+  ],
+};
+
+// A function to convert a document to form values.
+const docToFormValues = aDoc => ({
+  // The parseMoneyValue function will turn a MoneyValue into a
+  // value the LocalizedMoneyInput component can handle ({currency: amount})
+  prices: LocalizedMoneyInput.parseMoneyValues(aDoc.prices),
+});
+
+// a function to convert form values back to a document
+const formValuesToDoc = formValues => ({
+  // The convertToMoneyValue function will turn a LocalizedMoneyInput
+  // value into a value the API can handle
+  // It automatically converts to centPrecision or highPrecision
+  // depending on the number of supplied fraction digits and the
+  // used currency code.
+  // If you want to forbid highPrecision, then the form's validation
+  // needs to add an error when it sees a highPrecision price.
+  // See example below
+  prices: LocalizedMoneyInput.convertToMoneyValues(formValues.prices),
+});
+
+const validate = formValues => {
+  const errors = { prices: {} };
+  Object.keys(formValues.prices).forEach(currency => {
+    errors.prices[currency] = {};
+  })
+  const emptyCurrencies = LocalizedMoneyInput.getEmptyCurrencies(formValues.prices);
+  // ['ERU', 'USD']
+  // THis form dosen't accept highprecision prices
+  const highPrecisionCurrencies = LocalizedMoneyInput.getHighPrecisionCurrencies(formValues.prices);
+  // ['CAD', 'USD']
+  emptyCurrencies.forEach(emptyCurrency => {
```

Comment:
The spacing in this example is a bit weird. We do

```
get empty currency errors
get high precision currency errors
add empty currency errors

add high precision currency errors
```

instead of

```
get empty currency errors
get high precision currency errors

add empty currency errors
add high precision currency errors
```

or

```
get empty currency errors
add empty currency errors

get high precision currency errors
add high precision currency errors
```